### PR TITLE
Creates CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by asamborski**
This is the initial commit of the contributing guidelines. There are no issues or fixes associated with this PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ 3. There are no obvious errors, according to `go vet`.
+
+## Testing OpenPubkey Locally
+
+To build OpenPubkey, ensure you have Go version `>= 1.20` installed. To verify which version you have installed, try `go version`.
+
+To run the [Google example](https://github.com/openpubkey/openpubkey/tree/main/examples/google):
+ 1. Navigate to the `examples/google/` directory. 
+ 2. Execute `go build`
+ 3. Execute `google login` to generate a valid PK token using Google as your OIDC provider.
```
Comment (whalelines) [inline CONTRIBUTING.md:29]:
```suggestion
 3. Execute `./google login` to generate a valid PK token using Google as your OIDC provider.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ 4. Execute `google sign` to use the PK token generated in (3) to sign a verifiable message.
```
Comment (whalelines) [inline CONTRIBUTING.md:30]:
```suggestion
 4. Execute `./google sign` to use the PK token generated in (3) to sign a verifiable message.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+# Current Committers
+
+The Committers of OpenPubkey are:
+1. Ethan Heilman (@EthanHeilman)
+2. Jonny Stoten (@jonnystoten)
+3. Lucie Mugnier (@lgmugnier)
```
Comment (whalelines) [inline CONTRIBUTING.md:37]: Perhaps this should move below the description of the roles.

**Review by whalelines — COMMENTED** [review]
Often contributing guidelines include instructions on how to commit code, i.e., the commit must be "signed off", e.g., https://github.com/anchore/syft/blob/main/CONTRIBUTING.md#sign-off-your-work

**Comment by jonnystoten** [comment]
> Often contributing guidelines include instructions on how to commit code, i.e., the commit must be "signed off",

Good point, we need to require DCO sign-off I believe.

File: CONTRIBUTING.md (hunk 1 above)
Comment (asamborski) [reply CONTRIBUTING.md]: Fixed: [24d1cfd](https://github.com/openpubkey/openpubkey/pull/55/commits/24d1cfd78a82effcbe21cb05e91b97f2da325d40)

File: CONTRIBUTING.md (hunk 2 above)
Comment (asamborski) [reply CONTRIBUTING.md]: Fixed: [24d1cfd](https://github.com/openpubkey/openpubkey/pull/55/commits/24d1cfd78a82effcbe21cb05e91b97f2da325d40)

File: CONTRIBUTING.md (hunk 3 above)
Comment (asamborski) [reply CONTRIBUTING.md]: Modified: [24d1cfd](https://github.com/openpubkey/openpubkey/pull/55/commits/24d1cfd78a82effcbe21cb05e91b97f2da325d40)

**Comment by asamborski** [comment]
Added in licensing and sign off requirements.

**Review by jonnystoten — APPROVED** [review]
LGTM

**Review by EthanHeilman — APPROVED** [review]

**Review by lgmugnier — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Welcome to OpenPubkey! We are so excited you are here. Thank you for your interest in contributing your time and expertise to the project. The following document details contribution guidelines.
+
+# Getting Started
+
+Whether you're addressing an open issue (or filing a new one), fixing a typo in our documentation, adding to core capabilities of the project, or introducing a new use case, anyone from the community is welcome here at OpenPubkey.
+
+## Include Licensing at the Top of Each File
+At the top of each file in your commit, please ensure the following is captured in a comment: 
+
+``` SPDX-License-Identifier: Apache-2.0 ```
```
Comment (whalelines) [inline CONTRIBUTING.md:12]: Are you trying to make a code block or use a fixed-width font here? If you are trying to make a code block, the triple backticks need to appear alone on a line. If you want fixed-width font, just use single ticks.

**Review by whalelines — COMMENTED** [review]
Small formatting issue

File: CONTRIBUTING.md (hunk 4 above)
Comment (asamborski) [reply CONTRIBUTING.md]: Fixed! Thanks for catching that. : [07f484b](https://github.com/openpubkey/openpubkey/pull/55/commits/07f484bc3a428c4be89a5e2bbec3414703b183f5)

**Review by lgmugnier — APPROVED** [review]

**Pull request merged**